templates/vhost/_scriptalias.epp

```diff
@@ -0,0 +1,15 @@
+<%- $aliases = $scriptaliases -%>
```

Comment:
Note that scriptaliases is deprecated anyway. There's going to be a major release, so I'd first clean that up. https://github.com/puppetlabs/puppetlabs-apache/pull/2469 does that.

Reply:
@ekohl  resolved!  please review